walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -2,26 +2,33 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Stanley 🐈</title>
+    <title>Jonathan Aner's Portfolio</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
     <div id="content">
-      <h1>Stanley 🐈</h1>
-      <p>This is Stanley.</p>
-      <img src="images/stanley-1.jpg"/>
+      <h1>Jonathan Aner's Portfolio</h1>
+      <p>Welecome to my Portfolio</p>
 
-      <p>She is very ferocious!</p>
-      <img src="images/stanley-2.jpg"/>
+      <p><b>What makes Me Me?</b> </p>
+      <img src="my-images/about-me/jonathan-mainpage.jpg"/> 
+      <p>Press "Click Me" to see a blog post about me</p>
```

Comment:
Suggestion: you could rewrite this block like this:

```
<p><a href="...">Click here</a> to see a blog post about me</p>
```

Alternatively, the word "Click" might not be needed because it's a well established convention that hyperlink are clickable. We could have a more natural text such as `Check out this blog post about me`, or `See a blog post about me`.